sor/src/test/java/com/bazaarvoice/emodb/sor/core/SorUpdateTest.java

```diff
@@ -71,17 +71,32 @@ public void testUpdatesAndDatabusEvents() {
                 ImmutableSet.of("ignore"));
 
         // Verify that the ignorable flag is set
-        expectedUpdateIntentEvent = new UpdateIntentEvent(_dataStore, Lists.newArrayList(new UpdateRef("test:table", "rowkey", changeId2, ImmutableSet.of("ignore"))));
-        assertEquals(_updateIntentEvent.getUpdateRefs(), expectedUpdateIntentEvent.getUpdateRefs(), "Expected events not generated on databus");
+        assertEquals(_updateRefs,
+                Lists.newArrayList(new UpdateRef("test:table", "rowkey", changeId2, ImmutableSet.of("ignore"))),
+                "Expected events not generated on databus");
     }
 
-    @Subscribe
-    public void simulateDatabusSubscribeMethod(UpdateIntentEvent event) {
-        // Capture the updateIntentEvent
-        _updateIntentEvent = event;
-    }
 
-    private void resetValues() {
-        _updateIntentEvent = null;
+    @Test
+    public void testFailedDatabus() {
+
+        _dataStore.update("test:table", "rowkey", TimeUUIDs.newUUID(), Deltas.mapBuilder()
+                        .update("test", Deltas.literal("foo"))
+                        .build(),
+                new AuditBuilder().setComment("This update should succeed").build());
+
+        _eventWriterRegistry.registerDatabusEventWriter(refs -> {
+            throw new RuntimeException();
+        });
+
+        try {
+            _dataStore.update("test:table", "rowkey", TimeUUIDs.newUUID(), Deltas.mapBuilder()
+                            .update("test", Deltas.literal("testValue"))
+                            .build(),
+                    new AuditBuilder().setComment("This update should fail").build());
+            fail();
+        } catch (RuntimeException e) {
+            assertEquals(_dataStore.get("test:table", "rowkey").get("test"), "foo");
```

Comment:
Minor:  Test is good, but maybe add message to the failure so it's more clear _why_ this test failed, something like "Record should not have been updated after databus event write failed".